proxies/live/apiproxy/targets/sync-wrap.xml

```diff
@@ -21,11 +21,14 @@
       <Step>
         <Name>SpikeArrest</Name>
       </Step>
+      <Step>
+        <Name>AssignMessage.AddSyncWaitHeader</Name>
+      </Step>
     </Request>
   </PreFlow>
   <LocalTargetConnection>
-    <APIProxy>sync-wrap-{{ APIGEE_ENVIRONMENT }}</APIProxy>
+    <APIProxy>sync-wrap-pr-21</APIProxy>
     <ProxyEndpoint>default</ProxyEndpoint>
-    <Path>/sync-wrap/{{ SERVICE_BASE_PATH }}</Path>
+    <Path>/sync-wrap-pr-21/{{ SERVICE_BASE_PATH }}</Path>
```

Comment:
On pr 21 of sync-wrap as timeouts are too low for our service, the feature branch bumps the maximum allowed timeout specified for X-Sync-Wait to 30000. Needs further investigation and sync-wrap updating in the future